src/Randomizer.CrossPlatform/Views/GenerationSettingsBasicPanel.axaml.cs

```diff
@@ -219,7 +219,36 @@ private void OpenMsuSelectionWindow(MsuRandomizationStyle? randomizationStyle)
 
     private void ImportSeedButton_OnClick(object? sender, RoutedEventArgs e)
     {
-        _generationSettingsWindowService?.ApplyConfig(Data.Basic.ImportString, true);
+        if (_generationSettingsWindowService == null)
+        {
+            return;
+        }
+
+        if (_generationSettingsWindowService.VerifyConfigVersion(Data.Basic.ImportString, out var versionMismatch))
+        {
+            _generationSettingsWindowService?.ApplyConfig(Data.Basic.ImportString, true);
+        }
+        else if (versionMismatch)
+        {
+            var messageWindow = new MessageWindow(new MessageWindowRequest()
+            {
+                Message = "The randomizer version of the import settings string does not match the current application's version. Using the seed number in this version may produce different results from the original seed. Do you want to continue?",
```

Comment:
This might be a lot more work than it's really worth, but it would be nice if you could include both version numbers here.

Probably just something like:

```
The randomizer version of the import settings string does not 
match the current application's version. Using the seed number 
in this version may produce different results from the original 
seed. Do you want to continue?

Seed version: 9.7.4
Your version: 9.8.0
```

Reply:
I think the main problem with that I can think of is that while a seed may be generated on a specific build, it's not necessarily only applicable for that build since the randomizer version is only updated when we change the logic.